hack/run_update_templates.sh

```diff
@@ -24,8 +24,6 @@ OUT_DIR=$(mktemp -d -t swck-templates.XXXXXXXXXX) || { echo "Failed to create te
 
```

Comment:
How about deleting the `run_update_templates.sh` as we don't need it anymore.

Reply:
Make sense